ouroboros-network/src/Ouroboros/Network/NodeToClient/Version.hs

```diff
@@ -1,11 +1,11 @@
 
 {-# LANGUAGE NamedFieldPuns #-}
 
-module Ouroboros.Network.NodeToClient.Version
-  ( NodeToClientVersion (..)
+module Ouroboros.Network.NodeToClient.Version (
+    NodeToClientVersion (..)
   , NodeToClientVersionData (..)
-  , nodeToClientVersionCodec
   , nodeToClientCodecCBORTerm
+  , nodeToClientVersionCodec
```

Comment:
@bartfrenk the networking team is using slightly different [style guide](https://github.com/input-output-hk/ouroboros-network/blob/master/docs/StyleGuide.md).  We decided to keep the current format of export list, because it's used almost everywhere in the network code base.

Reply:
```suggestion
module Ouroboros.Network.NodeToClient.Version
  ( NodeToClientVersion (..)
  , nodeToClientVersionCodec
  , NodeToClientVersionData (..)
  , nodeToClientCodecCBORTerm
```
that would be probably better order.